source/Conversation.cpp

```diff
@@ -523,3 +577,14 @@ void Conversation::AddNode()
 	nodes.emplace_back();
 	nodes.back().data.emplace_back("", nodes.size());
 }
+
+
+
+bool Conversation::HasCondition(const DataNode &node)
```

Comment:
As mentioned, functions should appear in the same order as in the header, so this should be moved to below LoadGotos.